### ISSUE TITLE
shed_init produces yaml with Pythonic types described

### ISSUE THREAD

**Issue by martenson** [galaxyproject/planemo#243]
``` planemo shed_init --name=seqtk_test --owner=planemo --description="seqtk wrapper for test"```

produces

```
description: !!python/unicode 'seqtk wrapper for test'
name: !!python/unicode 'seqtk_test'
owner: !!python/unicode 'planemo'
```

it still works with other Planemo commands but maybe is not valid .yaml and the types are not neccesary I think.

**Comment by erasche** [comment]
oooh, yeah, those need to be cast in `str()`

**Comment by jmchilton** [comment]
Closed with f0610d7.
